scss/_patterns_side-navigation.scss

```diff
@@ -542,7 +544,7 @@
   // border color of items list
   $color-sidenav-list-border
 ) {
-  // stylelint-disable selector-no-qualifying-type
+  // stylelint-disable selector-no-qualifying-type, selector-max-type
```

Comment:
I wonder if we need `selector-no-qualifying-type` now, `selector-max-type` with value of 0 seems to catch all of these as well.

Reply:
Good points about the comments, adding more now. I did check whether we needed `selector-no-qualifying-type`, and we do, in every instance I remove it, it raises an error.

`selector-max-type` catches any instance of an html selector, which this is an instance of, but `<selector>.class`/`::psuedo`/`[attribute]`/`etc` is not caught by the same rule.

Reply:
@bartaz just saw your latest comment, I think my reply here got lost amongst the comments - we do still need the `selector-no-qualifying-type`.